rllib/examples/rnnsac_stateless_cartpole.py

```diff
@@ -18,8 +18,8 @@
     "keep_checkpoints_num": 1,
     "checkpoint_score_attr": "episode_reward_mean",
     "stop": {
-        "episode_reward_mean": 65.0,
-        "timesteps_total": 50000,
+        "episode_reward_mean": 80.0,
+        "timesteps_total": 500000,
```

Comment:
Hmm, do we have time to run 500k ts for this test? I'm sure this'll timeout all the time. The max we have is 15min.

Reply:
Just trying to make sure we didn't add a severe learning regression here (learning before in 50k ts, now requiring 500k).

Reply:
Hi, the 500k are a typo, sorry. The script learn really fast on my machine and so I could test around a little with it.
I'm changing it back and make sure it learns.